fighters/fox/src/acmd/specials.rs

```diff
@@ -100,40 +100,74 @@ unsafe fn sound_specialhi(fighter: &mut L2CAgentBase) {
 unsafe fn game_speciallwstart(fighter: &mut L2CAgentBase) {
     let lua_state = fighter.lua_state_agent;
     let boma = fighter.boma();
-    frame(lua_state, 1.0);
+    frame(lua_state, 0.0);
```

Comment:
Does a motion rate on frame 0 produce different results than putting it on frame 1? Test this and get back to me
Also once again people are putting `FT_MOTION_RATE` in `is_excute`, don't do that

Reply:
Tested this, yeah didn't need to be on F0, behaves the same with the call on F1
Please refer to my logic in the latest comment though, I'd like to know if i even need to do this at all